src/orange/OrangeTypes.hh

```diff
@@ -264,6 +264,14 @@ enum OperatorToken : logic_int
     ltrue,  //!< Push 'true'
     lend
 };
+
+//! Defines the notation for logic expressions
+enum class LogicNotation
+{
+    postfix,
+    infix,
+    size_
+};
```

Comment:
Since this is an enum class, we could just put it in the main namespace (and might make some of the JSON code easier too). The `OperatorToken` above is a non-class mainly for historical reasons.